benchmarks/dynamo/pr_time_benchmarks/check_results.py

```diff
@@ -92,34 +99,52 @@ def log(event_name):
                 ),
             )
 
+        ratio = float(result - entry.expected_value) * 100 / entry.expected_value
+
         if result > high:
+            new_entry = copy.deepcopy(entry)
+            new_entry.expected_value = result
+            new_expected[key] = new_entry
+
             fail = True
-            ratio = float(result - entry.expected_value) * 100 / entry.expected_value
             print(
                 f"REGRESSION: benchmark {key} failed, actual result {result} "
                 f"is {ratio:.2f}% higher than expected {entry.expected_value} ±{entry.noise_margin*100:.2f}% "
-                f"if this is an expected regression, please update the expected results."
+                f"if this is an expected regression, please update the expected results.\n"
             )
 
             log("fail_regression")
 
-        if result < low:
+        elif result < low:
+            new_entry = copy.deepcopy(entry)
+            new_entry.expected_value = result
+            new_expected[key] = new_entry
+
             fail = True
-            ratio = float(entry.expected_value - result) * 100 / entry.expected_value
 
             print(
                 f"WIN: benchmark {key} failed, actual result {result} is {ratio:.2f}% lower than "
                 f"expected {entry.expected_value} ±{entry.noise_margin*100:.2f}% "
-                f"please update the expected results."
+                f"please update the expected results.\n"
             )
 
             log("fail_win")
 
+        else:
+            ratio = float(entry.expected_value - result) * 100 / entry.expected_value
+
+            print(
+                f"PASS: benchmark {key} pass, actual result {result} {'+' if ratio>0 else ''}{ratio:.2f}% is within "
```

Comment:
You can force the sign by adding `+` to the format:
```suggestion
                f"PASS: benchmark {key} pass, actual result {result} {ratio:+.2f}% is within "
```